sycl/test-e2e/Matrix/SG32/element_wise_all_ops.cpp

```diff
@@ -6,7 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: aspect-ext_intel_matrix
-// REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
+// REQUIRES-INTEL-DRIVER: lin: 30049, win: 101.4943
```

Comment:
I believe, this one is failing for a different reason.
Instead of driver version update, it should be:
```
// TODO: Reenable, see https://github.com/intel/llvm/issues/14598
// UNSUPPORTED: windows, linux
```
Please, see example here: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/element_wise_all_ops.cpp
And this test should be mentioned in https://github.com/intel/llvm/issues/14598. It was not discovered before, because we did not have PVC runs and SG32 tests cannot run on ARC.